src/h5web/visualizations/heatmap/store.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-param-reassign */
+import { createContextStore, Action, State, action, Actions } from 'easy-peasy';
+import { ColorMap } from './interpolators';
+
+interface HeatmapState {
+  colorMap: ColorMap;
+  setColorMap: Action<HeatmapState, ColorMap>;
+}
+
+export const HeatmapStore = createContextStore<HeatmapState>({
+  colorMap: 'Magma',
+  setColorMap: action((state, colorMap) => {
+    state.colorMap = colorMap;
```

Comment:
We mutate the state. easy-peasy makes this operation safe, as explained here: https://easy-peasy.now.sh/docs/tutorial/using-actions-to-update-state.html -- unfortunately, ESLint doesn't like this, which is why I disable a rule at the top of the file.